com.github.IsmaelMartinez.teams_for_linux.appdata.xml

```diff
@@ -14,6 +14,15 @@
 	<url type="bugtracker">https://github.com/IsmaelMartinez/teams-for-linux/issues</url>
 	<launchable type="desktop-id">com.github.IsmaelMartinez.teams_for_linux.desktop</launchable>
 	<releases>
+		<release version="2.5.9" date="2025-09-19">
+			<description>
+				<ul>
+					<li>Feature: Implement secure token cache storage using Electron safeStorage API - tokens are now encrypted using OS-level security (Keychain/DPAPI/kwallet) instead of plain localStorage</li>
+					<li>Enhancement: Add automatic migration from localStorage to secure storage with fallback protection for older systems</li>
+					<li>Security: Improve authentication persistence security while maintaining compatibility across platforms</li>
```

Comment:
```suggestion
					<li>Feature: Implement secure token cache storage using Electron safeStorage API - tokens are now encrypted using OS-level security (Keychain/DPAPI/kwallet) instead of plain localStorage</li>
```